.github/workflows/test.yml

```diff
@@ -1,35 +1,42 @@
 name: Test
 
 on:
-  push:
-    branches:
-      - "master"
-      - "develop"
-  pull_request:
-    types: [ready_for_review, synchronize, opened]
+    push:
+        branches:
+            - "master"
+            - "develop"
+    pull_request:
+        types: [ ready_for_review, synchronize, opened ]
 
 jobs:
     unit:
         runs-on: ubuntu-latest
         strategy:
             matrix:
                 # test against latest update of each major Java version, as well as specific updates of LTS versions:
-                java: [ 8, 9, 10, 11, 12, 13 ]
+                java: [ 8, 11, 15, 16 ]
```

Comment:
@faustbrian I think we should only support LTS versions: those are 8, 11 and 17 is coming up in a couple months. So supporting the 15 and 16 seems a good idea before 17 is out.